ofl/agbalumo/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2023 The Danfo Project Authors (https://github.com/googlefonts/googlefonts-project-template)
```

Comment:
Missing correct URL and copyright string needs be updated with the new name.

Reply:
This was fixed in this upstream PR: https://github.com/SorkinType/Agbalumo/pull/17